TOMP-API.yaml

```diff
@@ -1784,65 +1866,83 @@ components:
         name:
           description: Human readable name of the place
           type: string
-        stop-reference:
+        stopReference:
           type: array
           items:
-            $ref: '#/components/schemas/stop-reference'
-        station-id:
+            $ref: '#/components/schemas/stopReference'
+        stationId:
           description: reference to /operator/stations
           type: string
-        coordinates:
-          $ref: '#/components/schemas/coordinates'
-        extra-info:
+        coordinate:
+          $ref: '#/components/schemas/coordinate'
+        extraInfo:
           type: array
           items:
-            $ref: '#/components/schemas/key-value'
-        physical-address:
+            $ref: '#/components/schemas/keyValue'
+        physicalAddress:
           $ref: '#/components/schemas/address'
- 
-    station-information:
+
+    stationInformation:
+      type: object
+      required:
+        - stationId
+        - name
+        - coordinate
       properties:
-        stations:
+        stationId:
+          type: string
+          description: unique identifier of a station
+          example: XX:Y:12345678
+        name:
+          type: string
+          description: public name of the station
+          example: Island Central
+        coordinate:
+          $ref: '#/components/schemas/coordinate'
+        physicalAddress:
+          $ref: '#/components/schemas/address'
+        crossStreet:
+          type: string
+          description: Cross street of where the station is located. This field is intended to be a descriptive field for human consumption. In cities, this would be a cross street, but could also be a description of a location in a park, etc.
+          example: on the corner with Secondary Road
+        regionId:
+          type: string
+          description: ID of the region where the station is located (see "systemRegions")
+        serviceArea:
```

Comment:
This just duplicates the functionality of systemRegion.

Reply:
don't think so. The region is for example 'Amsterdam' (where they are allowed to operate). Stations can have their own 'subregions' within this region.

Reply:
One system can have multiple system regions, so that could include those subregions.

Station information contains the regionId of the region it operates in. What is the distinction you are trying to make between a station's (own) region and its service area?